tests/installation/partitioning_resize_root.pm

```diff
@@ -32,7 +32,7 @@ sub run {
     send_key_until_needlematch 'volume-management-root-selected', 'down';
     send_key $cmd{resize};
     assert_screen 'volume-management-resize-maximum-selected';
-    send_key $cmd{ok};
+    send_key(check_var('VERSION', '15') ? $cmd{next} : $cmd{ok});
```

Comment:
please don't. This would make the test fail as soon as SLE 15 SP1. Look at L24. We use `is_storage_ng`